tests/foreman/cli/test_variables.py

```diff
@@ -1014,14 +1313,22 @@ def test_positive_unhide_default_value(self):
         3. After hiding, set '--hidden-value' to false.
 
         @assert: The default value is set to false.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
         """
+        smart_variable = make_variable({
+            'variable': gen_string('alpha'),
+            'puppet-class': self.puppet['name'],
+            'default-value': gen_string('alpha'),
+            'hidden-value': 1,
+        })
+        self.assertEqual(smart_variable['hidden-value?'], True)
+        SmartVariable.update({
+            'name': smart_variable['name'],
+            'hidden-value': 0
+        })
+        updated_sv = SmartVariable.info({'name': smart_variable['name']})
+        self.assertEqual(updated_sv['hidden-value?'], False)
```

Comment:
i believe it was supposed to only return `hidden_value: *****` if `hidden_value` is `True`. So I would file a bug for this